File: python/luisdomiciano/area-of-rectangle/area_of_a_rectangle.py
def area(side1, side2):
    if (side1 <= 0 or side2 <= 0):
        return -1
    else:
        return side1 * side2

print (area(3, 4))
print (area(10, 11))
print (area(-1, 5))
print (area(0, 2))

File: javascript/guilhermescr/DesafioQuatro/areaOfARectangle.js
function area(n1, n2) {
  if (n1 <= 0 || n2 <= 0) {
    return console.log(-1);
  }

  let rectangleArea = n1 * n2;

  return console.log(rectangleArea);
}

area(3, 4);
area(10, 11);
area(-1, 5);
area(0, 2);
